test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -612,13 +612,16 @@ class TestDapp {
    * @param options - Options for connecting account to test dapp.
    * @param [options.connectAccountButtonEnabled] - Indicates if the connect account button should be enabled.
    * @param options.publicAddress - The public address to connect to test dapp.
+   * @param [options.networkId] - The network to connect to, defaults to 0x539.
```

Comment:
mm I think we might want to reference the chainId here?

```suggestion
   * @param [options.chainId] - The chain id to connect to, defaults to 0x539.
```

![Screenshot from 2025-02-04 18-20-27](https://github.com/user-attachments/assets/92180dd7-e86b-44c8-b2f3-8913d9bfd000)

Reply:
yes make sense, i wanted to refer to chainId, thanks for the good catch !